docs/README.md

```diff
@@ -70,6 +70,8 @@ onCameraDidStopRunning | func | no |  | Called when the camera has stopped runin
 onStatsReceived | func | no |  | Called when stats are received (after calling getStats)
 onNetworkQualityLevelsChanged | func | no |  | Called when the network quality levels of a participant have changed (only if enableNetworkQualityReporting is set to True when connecting)
 onDominantSpeakerDidChange | func | no |  | Called when dominant speaker changes @param {{ participant, room }} dominant participant
+onLocalParticipantH246Supported | func | no |  | Called when an android device does not have H264 support
```

Comment:
Since it is called `onLocalParticipantH246Supported`, I think it should always be called and send `true` or `false`.

If you want to trigger the event ONLY when it is not supported, then `onLocalParticipantH246NotSupported` would be a better name.